.github/workflows/project_ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       # Instantiate the project
       - name: Instantiate project using the values in cookiecutter.json
-        run: poetry run cookiecutter . --no-input
+        run: poetry run cookiecutter . --no-input --accept-hooks no
```

Comment:
what does this change do?

Reply:
Stops the post-gen hook from running. Keeps us from having to do all of the package setup twice, saving ~3-4 minutes per test run.